packages/react-components/react-portal-compat-context/tsconfig.lib.json

```diff
@@ -2,7 +2,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "noEmit": false,
-    "lib": ["ES2019", "dom"],
+    "lib": ["ES5", "ES2015.Iterable", "dom"],
```

Comment:
same changes for `packages/react-components/react-portal-compat-context/tsconfig.json` as well 🙏